velox/docs/functions/presto/tdigest.rst

```diff
@@ -0,0 +1,107 @@
+==================
+T-Digest Functions
+==================
+
+Velox implements algorithms for estimating rank-based metrics using `T-digest
+<https://doi.org/10.1016/j.simpa.2020.100049>`_. T-digest has `better
+performance <https://arxiv.org/abs/1902.04023>`_ in general and has
+better accuracy at the tails, often dramatically better, but may have worse
+accuracy at the median, depending on the compression factor used.
+
+T-digest was developed by Ted Dunning.
+
+Data Structures
+---------------
+
+A T-digest is a data sketch which stores approximate percentile information.
+The Velox type for this data structure is called ``tdigest``,
+and it accepts a parameter of type ``double`` which represents the set of
+numbers to be ingested by the ``tdigest``. Other numeric types may be added
+in a future release.
+
+T-digests may be merged without losing precision, and for storage and retrieval
+they may be cast to/from ``VARBINARY``.
+
+Functions
+---------
+
+.. function:: tdigest_agg(x) -> tdigest
+
+    Returns the ``tdigest`` which summarizes the approximate distribution of all input values of ``x``.
+    The default compression factor is ``100``.
+
+.. function:: tdigest_agg(x, w) -> tdigest
+   :noindex:
+
+    Returns the ``tdigest`` which summarizes the approximate distribution of all input values of ``x`` using per-item weight ``w``.
+    The default compression factor is ``100``.
+
+.. function:: tdigest_agg(x, w, compression) -> tdigest
+   :noindex:
+
+    Returns the ``tdigest`` which summarizes the approximate distribution of all input values of ``x`` using per-item weight ``w`` and the specified compression factor.
+    ``compression`` must be greater than or equal to ``1``. Higher compression means more accuracy at the cost of more memory.
```

Comment:
Does compression need to be constant or just constant for a given group?

Reply:
It needs to be constant, specified that as long as more details around acceptable values